pom.xml

```diff
@@ -71,7 +71,7 @@
   <properties>
     <revision>2.71</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.204.6</jenkins.version>
+    <jenkins.version>2.222.1</jenkins.version>
```

Comment:
We will have to take care to highlight the pre-requisite dump (For example I will have to do it in Advisor too)

Reply:
The was the only way to leverage the `GlobalConfigurationCategory`. What else need to be done to highlight the pre req ?